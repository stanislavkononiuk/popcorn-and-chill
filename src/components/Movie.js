import React from 'react';
import PropTypes from 'prop-types';
import MovieCSS from './Movie.module.css';

const Movie = ({ imageScr, score, title }) => (
  <div className={MovieCSS.movieWraper}>
    <img className={MovieCSS.image} src={imageScr} alt={title} />
    <div className={MovieCSS.infoWraper}>
      <h2 className={MovieCSS.header}>{title}</h2>
      <p className={MovieCSS.score}>
        score:
        {' '}
        {score}
      </p>
    </div>
  </div>
);

Movie.propTypes = {
  imageScr: PropTypes.string.isRequired,
  score: PropTypes.number.isRequired,
  title: PropTypes.string.isRequired,
};

export default Movie;
