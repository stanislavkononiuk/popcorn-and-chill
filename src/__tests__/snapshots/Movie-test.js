import React from 'react';
import renderer from 'react-test-renderer';
import Movie from '../../components/Movie';

it('renders correctly the Details component', () => {
  const tree = renderer
    .create(<Movie title="popcorn-n-chill" imageScr="my image" score={3} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
