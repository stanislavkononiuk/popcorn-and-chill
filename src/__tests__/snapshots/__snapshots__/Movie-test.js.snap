// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly the Details component 1`] = `
<div
  className="movieWraper"
>
  <img
    alt="popcorn-n-chill"
    className="image"
    src="my image"
  />
  <div
    className="infoWraper"
  >
    <h2
      className="header"
    >
      popcorn-n-chill
    </h2>
    <p
      className="score"
    >
      score:
       
      3
    </p>
  </div>
</div>
`;
